RxMarbles/HelpViewController.swift

```diff
@@ -551,7 +553,17 @@ class HelpViewController: AnimatedPagingScrollViewController {
     }
     
     private func _configureSharePage() {
-        let navBar = RxMarbles.Image.navBarShare.imageView()
+        var navBar = UIImageView()
+        if #available(iOS 13.0, *) {
+            if self.traitCollection.userInterfaceStyle == .dark {
```

Comment:
@Gtwatt1 it is better to use assets appearance and not introduce new var (`navBarShareDark`)
<img width="277" alt="image" src="https://user-images.githubusercontent.com/5250/72322252-f1638380-36b6-11ea-8fb0-768c77637fc9.png">

Reply:
thanks and it has been fixed.